geowebcache/core/src/main/java/org/geowebcache/storage/blobstore/file/FileBlobStore.java

```diff
@@ -324,7 +330,7 @@ public boolean delete(TileObject stObj) throws StorageException {
             if (!fh.delete()) {
                 throw new StorageException("Unable to delete " + fh.getAbsolutePath());
             }
-            stObj.setBlobSize((int) length);
+            stObj.setBlobSize((int) padSize(length));
```

Comment:
I'm a bit worried about something calling `getBlobSize()` and not knowing if it's getting the size of the data, or the size of the space in storage that it consumes.